Sources/SpeziOnboarding/SignatureView.swift

```diff
@@ -26,7 +26,7 @@ import SwiftUI
 /// )
 /// ```
 public struct SignatureView: View {
-    @Environment(\.undoManager) private var undoManager
+    @Environment(\.undoManager) private var undoManager // swiftlint:disable:this attributes
```

Comment:
Similar to https://github.com/StanfordSpezi/SpeziHealthKit/pull/7, feel free to change the SwiftLint file to remove the rule here.